Cql/Cql.Compiler/ExpressionBuilder.cs

```diff
@@ -2540,7 +2540,7 @@ protected Expression ParameterRef(ParameterRef op, ExpressionBuilderContext ctx)
 
         }
 
-        protected MemberInfo GetProperty(Type type, string name)
+        protected internal MemberInfo GetProperty(Type type, string name)

```

Comment:
Any reason this is opened up to internal? I noticed the test, but perhaps it could be put on the TypeResolver itself.